### ISSUE TITLE
Event Attendance 

### ISSUE THREAD

**Issue by nazmicancalik** [bounswe/bounswe2018group9#258]
Event attendance endpoint is required for registering users to the events. 

POST to: __/api/events/:id/attendance__

`req.body: {  
attendant: user_id,
attendanceType: 0 | 1 | 2
}
`

**0 => will not attend                                
1 => going to attend                                
2 => may attend**

This is the format I have come up with, any feedback or improvements are welcome. @okyksl @serkan-ozel-2015400123 @gumityolcu

**Comment by okyksl** [comment]
Should we also include the following?

**3 => attended**
**4 => not attended**

**Comment by gumityolcu** [comment]
@nazmicancalik 
Hello,

I reviewed the code, it seemed fine to me. But when I tested I got the error "Invalid select() argument. Must be string or object.". I tried to resolve the issue but couldn't put my finger on it.

Test request that I used:
POST http://127.0.0.1:3000/api/events/5bd820ac7c9ab2002fe3bb2c/attendee

Header:
Content-Type: application/json
Body:
{
  "attendant" : "5bd8143a7c9ab2002fe3bb29", 
  "attendanceType" : "1"
}
